src/OpenCOVER/vrml97/js/jsopcode.c

```diff
@@ -1020,7 +1020,7 @@ Decompile(SprintStack *ss, jsbytecode *pc, intN nb)
         op = saveop = (JSOp)*pc;
         if (op >= JSOP_LIMIT)
         {
-            switch (op)
```

Comment:
perhaps this comparison should be done before casting *ps to JSop? Why should it be save to use the value of *pc as JSOp afterwards?